client/account-recovery/lost-password/test/index.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import reactTapEventPlugin from 'react-tap-event-plugin';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { LostPassword } from '..';
+
+describe( 'LostPassword', () => {
+	it( 'should render as expected', () => {
+		const wrapper = shallow( <LostPassword className="test__test" translate={ noop } /> );
+
+		expect( wrapper ).to.have.className( 'lost-password__container' );
+		expect( wrapper ).to.have.className( 'test__test' );
+		expect( wrapper ).to.have.state( 'isSubmitting' ).to.be.false;
+		expect( wrapper.find( '.lost-password__user-login-input' ).prop( 'disabled' ) ).to.not.be.ok;
```

Comment:
`enzyme-chai` convenience for this could be:

```js
expect( wrapper.find( '.lost-password__user-login-input' ) ).to.not.have.prop( 'disabled' );
```